tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier1,
+    polarion_id,
+)
+from ocs_ci.ocs.exceptions import (
+    CommandFailed,
+    TimeoutExpiredError,
+    UnexpectedBehaviour,
+)
+from ocs_ci.ocs.resources.pod import get_file_path, check_file_existence
+from ocs_ci.helpers.helpers import fetch_used_size
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@skipif_ocs_version("<4.10")
+class TestRbdSpaceReclaim(ManageTest):
+    """
+    Tests to verify RBD space reclamation
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, project_factory, storageclass_factory, create_pvcs_and_pods):
+        """
+        Create PVCs and pods
+        """
+        self.pool_replica = 3
+        pvc_size_gi = 25
+        self.sc_obj = storageclass_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            replica=self.pool_replica,
+            new_rbd_pool=False,
+        )
+        self.pvc, self.pod = create_pvcs_and_pods(
+            pvc_size=pvc_size_gi,
+            access_modes_rbd=[constants.ACCESS_MODE_RWO],
+            num_of_rbd_pvc=1,
+            num_of_cephfs_pvc=0,
+            sc_rbd=self.sc_obj,
+        )
+
+    @polarion_id("OCS-2759")
+    @tier1
+    def test_rbd_space_reclaim_cronjob(self):
+        """
+        Test to verify RBD space reclamation
+        Steps:
+        1. Create and attach RBD PVC of size 25 GiB to an app pod.
+        2. Get the used size of the RBD pool
+        3. Create four files of size 4GiB
+        4. Verify the increased used size of the RBD pool
+        5. Delete three file
+        6. Create ReclaimSpaceJob
+        7. Verify the decreased used size of the RBD pool
+        8. Verify the presence of other files in the folder
+        """
+        pvc_obj = self.pvc[0]
+        pod_obj = self.pod[0]
+
+        fio_filename1 = "fio_file1"
+        fio_filename2 = "fio_file2"
+        fio_filename3 = "fio_file3"
+        fio_filename4 = "fio_file4"
+
+        schedule = ["hourly", "midnight", "weekly"]
+        # Fetch the used size of pool
+        cbp_name = self.sc_obj.get().get("parameters").get("pool")
+        log.info(f"Cephblock pool name {cbp_name}")
+        used_size_before_io = fetch_used_size(cbp_name)
+        log.info(f"Used size before IO is {used_size_before_io}")
+
+        # Create four 4 GB file
+        for filename in [fio_filename1, fio_filename2, fio_filename3, fio_filename4]:
+            pod_obj.run_io(
+                storage_type="fs",
+                size="4G",
+                runtime=100,
```

Comment:
as Avi mentioned in PR #5936 in given time 4G data will not be written to the PVC.